docs/en/get_started/build.md

```diff
@@ -74,41 +75,41 @@ You should know how to set up environment variables, especially `Path`, on Windo
 
 1. Launch Anaconda prompt from Windows Start menu
 
-    Do not use raw `cmd.exe` s instruction is based on PowerShell syntax.
+   Do not use raw `cmd.exe` s instruction is based on PowerShell syntax.
```

Comment:
we may check whether the indentation is expected after modification

Reply:
It works as expected.